code/game/objects/items/miscellaneous.dm

```diff
@@ -337,6 +337,28 @@
 	to_chat(M, msg)
 	new /obj/effect/pod_landingzone(get_turf(src), pod)
 
+/obj/item/choice_beacon/petchoice
+	name = "Multiple choice animal delivery beacon"
+	desc = "There are no faster ways, only more humane."
+	var/default_name = "Petsie"
+	
+/obj/item/choice_beacon/petchoice/generate_display_names()
```

Comment:
Where is this ever called?
Why not use initialize?

Reply:
i think it's the choice beacon proc for list selections, seems to be used by at least 1 other thing in this file such as the magician's tools

Reply:
musician quirk also uses a choice beacon.

Reply:
and curator